ui/src/layout/Menu.js

```diff
@@ -80,33 +96,91 @@ const Menu = ({ onMenuClick, dense, logout }) => {
     )
   }
 
-  const subItems = (subMenu) => (resource) =>
-    resource.hasList && resource.options && resource.options.subMenu === subMenu
+  const subItems = (subMenu) => (resource) => {
+    const { hasList, options, name } = resource
+    return hasList && name !== 'playlist' && options?.subMenu === subMenu
+  }
 
-  return (
-    <div>
-      <SubMenu
-        handleToggle={() => handleToggle('menuAlbumList')}
-        isOpen={state.menuAlbumList}
-        sidebarIsOpen={open}
-        name="menu.albumList"
-        icon={<AlbumIcon />}
-        dense={dense}
-      >
-        {Object.keys(albumLists).map((type) =>
-          renderAlbumMenuItemLink(type, albumLists[type])
-        )}
-      </SubMenu>
-      <SubMenu
-        handleToggle={() => handleToggle('menuLibrary')}
-        isOpen={state.menuLibrary}
+  const RenderPlaylistLinks = ({ playlist }) => {
+    const { id, name } = playlist
+    return (
+      <MenuItemLink
+        key={id}
+        to={`/playlist/${id}/show`}
+        primaryText={name}
+        onClick={onMenuClick}
         sidebarIsOpen={open}
-        name="menu.library"
-        icon={<LibraryMusicIcon />}
         dense={dense}
+      />
+    )
+  }
+
+  const classes = useStyles()
+  const refresh = useRefresh()
+  const [playLists, setPlaylists] = useState([])
+  const { data } = useGetList('playlist', { page: 1, perPage: -1 }, {}, {})
+
+  useEffect(() => {
+    const isEmpty = !Object.keys(data).length
+
+    if (!isEmpty) {
+      setPlaylists(Object.values(data))
+    } else if (isEmpty && playLists.length) {
+      refresh()
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data])
+
+  return (
+    <div className={classes.menuWrapper}>
+      <div>
+        <SubMenu
+          handleToggle={() => handleToggle('menuAlbumList')}
+          isOpen={state.menuAlbumList}
+          sidebarIsOpen={open}
+          name="menu.albumList"
+          icon={<AlbumIcon />}
+          dense={dense}
+        >
+          {Object.keys(albumLists).map((type) =>
+            renderAlbumMenuItemLink(type, albumLists[type])
+          )}
+        </SubMenu>
+        <SubMenu
+          handleToggle={() => handleToggle('menuLibrary')}
+          isOpen={state.menuLibrary}
+          sidebarIsOpen={open}
+          name="menu.library"
+          icon={<LibraryMusicIcon />}
+          dense={dense}
+        >
+          {resources
+            .filter(subItems('library'))
+            .map(renderResourceMenuItemLink)}
+        </SubMenu>
+      </div>
+      <div
+        style={{
+          overflowY: open && state.menuPlaylists ? 'scroll' : 'auto',
+          overflowX: 'hidden',
+        }}
       >
-        {resources.filter(subItems('library')).map(renderResourceMenuItemLink)}
-      </SubMenu>
+        <SubMenu
+          handleToggle={() => handleToggle('menuPlaylists')}
+          isOpen={state.menuPlaylists}
+          sidebarIsOpen={open}
+          name="resources.playlist.name"
```

Comment:
You can set `name` using `translatedResourceName` to get the proper translation for the resource name